pkg/config/derived-role-cert.go

```diff
@@ -43,29 +43,44 @@ func (idCfg *IdentityConfig) derivedRoleCertConfig() error {
 		return nil // disabled
 	}
 
-	if idCfg.roleCertDir == "" {
+	if idCfg.roleCertDir == "" && idCfg.roleCertNamingFormat == "" {
 		return nil // disabled
 	}
+	// If both the RoleCert settings and the NamingFormat settings are configured redundantly, an error will be returned.
+	if idCfg.roleCertDir != "" && idCfg.roleCertNamingFormat != "" {
+		return fmt.Errorf("RoleCertDir and RoleCertNamingFormat are both set: RoleCertDir %s, RoleCertNamingFormat %s", idCfg.roleCertDir, idCfg.roleCertNamingFormat)
+	}
+	if idCfg.roleCertDir != "" && idCfg.roleCertKeyNamingFormat != "" {
+		return fmt.Errorf("RoleCertDir and RoleCertKeyNamingFormat are both set: RoleCertDir %s, RoleCertKeyNamingFormat %s", idCfg.roleCertDir, idCfg.roleCertKeyNamingFormat)
+	}
```

Comment:
same above

Reply:
handled in: https://github.com/AthenZ/k8s-athenz-sia/pull/149/commits/8515d06e7f4342ddf4c18ab94b97774fa107e598